components/PrivacyPolicyModal.tsx

```diff
@@ -0,0 +1,140 @@
+import { Flex, Heading, Link, ListItem, Text, UnorderedList } from '@chakra-ui/layout';
+import { Modal, ModalBody, ModalCloseButton, ModalContent, ModalOverlay } from '@chakra-ui/modal';
+
+type PrivacyPolicyModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+export const PrivacyPolicy = () => {
+  return (
+    <Flex
+      className="container"
+      direction="column"
+      maxWidth="40rem"
+      marginTop="3rem"
+      marginBottom="3rem"
+    >
+      <Heading as="h1" size="lg" alignSelf="center">
```

Comment:
nit: think it's slightly improved w/ a little more header space here

```suggestion
      <Heading as="h1" size="lg" alignSelf="center" mb="1rem">
```